CMakeLists.txt

```diff
@@ -478,6 +478,10 @@ if(BUILD_TESTS)
       # purpose to check that a recovery network is robust to a view change.
       --raft-election-timeout
       4000
+      # Smaller ledger chunks than default to make sure that a ledger made of
+      # multiple chunks can be recovered
+      --ledger-chunk-threshold
```

Comment:
Can we make this --chunk-max-bytes, that way we're consistent with the --sig-max-* flags, and that leaves the ability to have --chunk-max-ms later.

Reply:
Went with `--ledger-chunk-max-bytes`.